src/pages/observability-packs.js

```diff
@@ -351,6 +352,24 @@ const ObservabilityPacksPage = ({ data, location }) => {
           `}
         `}
       >
+        <a
+          href="https://github.com/newrelic/newrelic-observability-packs"
+          target="_blank"
+          rel="noreferrer"
```

Comment:
I _think_ we might be able to use the `ExternalLink` component for this. Not sure if it's worth the update, just wanted to mention it.